### DISCUSSION TITLE
Add a property to AvatarView or make a converter.

### DISCUSSION THREAD

**Discussion by cl2raul66** [xamarin/XamarinCommunityToolkit#1618] [Ideas]
It would be very good if you add a true or false option, which allows you to convert the "Text" property to initials or to develop a text string to initials converter.

**Comment by AndreiMisiukevich** [comment]
We had some discussion around that, but it's not as simple as it seems at first glance.
We can develop a super simple algorithm of extracting initials from Full names, but it won't cover all possible cases.

e.g. how should we handle the next cases?

```María José Carreño Quiñones```
```毛泽东```
```Robert John Downey Jr.```
